src/main/java/org/jenkins/tools/test/maven/ExternalMavenRunner.java

```diff
@@ -67,7 +71,7 @@ public void run(Config config, File baseDirectory, File buildLogFile, String...
         }
         cmd.addAll(config.mavenOptions);
         cmd.addAll(Arrays.asList(goals));
-        System.out.println("running " + cmd + " in " + baseDirectory + " >> " + buildLogFile);
+        LOGGER.log(Level.INFO, "Running {0} in {1} >> {2}" + buildLogFile, new Object[]{String.join(" ", cmd), baseDirectory, buildLogFile});
```

Comment:
Removing the commas between arguments to make it easier to copy and paste the string to reproduce issues locally.

Reply:
Thanks, my eyes were glazing over by this point…